bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -140,6 +140,13 @@ except KeyError:
     dec_chirp_dist = pycbc.pnutils.chirp_distance(dec_dist, mchirp)
     sorting = dec_chirp_dist.argsort()  # ascending order of dec chirp distance
 
+# Get the trigger event times
+trigger_times = {}
+for trig in single_triggers:
+    ifo = trig.ifo
+    with h5py.File(trig.lfn, 'r') as trig_f:
```

Comment:
I think this will hit the same issue as noted in https://github.com/gwastro/pycbc/pull/4510

I will implement something similar to the fix there in this

Reply:
Actually this won't be as bad as we will be using the injection HDF_TRIGGER_MERGE files rather than FULL_DATA, but should still be fixed

Reply:
It's less important here, as the injection TRIGGER_MERGE are an order of magnitude smaller than the full_data one ... but still, avoiding having all trigger times read into memory, when it's not needed, could be useful. It would likely slow the code down a bit though, hopefully only a small amount.

Reply:
I've added a method to find the triggers in a +/- 2s window around the missed injections, and not to display triggers outwith the window.

It might be good to cut this default to 1s maybe